src/lib/reasoners/satml_frontend.ml

```diff
@@ -1001,33 +1001,10 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
       may_update_last_saved_model env compute_model;
       Options.Time.unset_timeout ();
       i_dont_know env unknown_reason
-    (* Timeout -> I_dont_know conversions temporarily disabled
-       https://github.com/OCamlPro/alt-ergo/issues/946 *)
-    with Util.Timeout when env.model_gen_phase && false ->
+    with Util.Timeout when env.model_gen_phase ->
```

Comment:
I believe that this PR removes the only piece of code that sets `model_gen_phase` to `true`, so the field should be removed.